### ISSUE TITLE
Chat UI Styling  - ZDCChatOverlay not styling 

### ISSUE THREAD

**Issue by hinduasish** [zendesk/zendesk_sdk_chat_ios#165]
- Xcode version: Latest
- iOS or Simulator: Simulator
- iOS Version:10.3
- Swift version:3

Hello, when I try to style ZDCChatOverlay, XCode tells me that zdcchatoverlay has no member messagecountcolor. it tells me it has methods like .tintcolor and .bordercolor yet they do not do anything. Any help with this would be appreciated.. I have attached the code that is causing the issue.

ZDCChatOverlay.appearance().tintColor = UIColor.red
ZDCChatOverlay.appearance().borderColor = UIColor.red
ZDCChatOverlay.appearance().MessageCountColor = UIColor.red
ZDCChat.start(in: self.navigationController, withConfig: nil)

.borderColor and .tintColor give me no compiling errors but do not do anything, i tried to add .MessageCountColor but it gives me a compiling error.

**Comment by RonanMcH** [comment]
Hi @hinduasish ,
`ZDCChatOverlay.appearance().messageCountColor = UIColor.red` compiles for on the latest version of the chatSDK. 
Some of the other styling options for the overlay we offer are:
```
ZDCChatOverlay.appearance().overlayTintColor = UIColor.red
ZDCChatOverlay.appearance().typingIndicatorColor = UIColor.red
```
More are detailed in this file [ZDCStyling](https://github.com/zendesk/zendesk_sdk_chat_ios/blob/master/ZopimChatSample/ZopimChatSample/ChatStyling.m) in the sample app. Unfortunately, they written in objective-c so you need to convert them to Swift following the pattern for the ones above.
Thnaks,
Ronan.

**Comment by hinduasish** [comment]
wow thank you for the quick reply, let me try again and i will let you know! maybe I am not using the latest version of the sdk

**Comment by hinduasish** [comment]
yes it works! looks like i was writing MessageCountColor instead of messageCountColor. Thank you so much, I appreciate it. How can i know which capitalisation to use when in ZopimChatSample/ChatStyling.m , the functions are written as setTypingIndicatorColor. Am I correct to assume that in swift all the functions will begin with lowercase letters and then capitalize every first letter of the word? Thanks regardless, closing issue....
